packages/commercetools/helpers/src/index.ts

```diff
@@ -28,6 +30,23 @@ export const getProductVariants = (products: ProductVariant[], options: any = {}
   return products
 }
 
+export const getProductAttributes = (product: ProductVariant, filterByAttributeName?: Array<string>): any => {
+  return (product ? formatAttributeList(product.attributeList) : [])
+  .map(a => ({
+    name: a.name,
+    value: a.value,
+    label: a.label ? a.label : a.value
+  }))
+  .filter(attribute => {
```

Comment:
Isn't better to use here ternary operator?

```TS
.filter(attribute => filterByAttributeName ? filterByAttributeName.includes(attribute.name) : attribute)
```
```

Reply:
imho its equally readable ;)